homeassistant/components/qbittorrent/coordinator.py

```diff
@@ -33,7 +40,19 @@ def __init__(self, hass: HomeAssistant, client: Client) -> None:
         )
 
     async def _async_update_data(self) -> dict[str, Any]:
+        """Async method to update QBittorrent data."""
         try:
             return await self.hass.async_add_executor_job(self.client.sync_main_data)
         except LoginRequired as exc:
             raise ConfigEntryError("Invalid authentication") from exc
+
+    async def get_torrents(self, torrent_filter: str) -> list[dict[str, Any]]:
+        """Async method to get QBittorrent torrents."""
+        try:
+            total_torrents = await self.hass.async_add_executor_job(
```

Comment:
`total_torrents` is a bit weird for something which is a list, `torrents` would be better.